jupyter-notebooks/gee-integration/gee-integration.ipynb

```diff
@@ -0,0 +1,324 @@
+     ]
+    }
+   ],
+   "source": [
+    "print(order_details['_links']['results'][0])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build an order request with clipping and harmonization applied\n",
+    "Here we are building an order request similar to the one above, but with [clipping](https://developers.planet.com/apis/orders/tools/#clip) and [harmonization](https://developers.planet.com/apis/orders/tools/#harmonization) applied. Namely, we are asking the Orders API to clip our images to the AOI, as defined by `iowa_aoi`, and radiometrically harmonizing the imagery with Sentinel-2's sensor."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'name': 'iowa_order', 'products': [{'item_ids': ['20200925_161029_69_2223', '20200925_161027_48_2223'], 'item_type': 'PSScene', 'product_bundle': 'analytic_sr_udm2'}], 'delivery': {'google_earth_engine': {'project': 'planet-devrel-dev', 'collection': 'gee-integration-testing'}}, 'tools': [{'harmonize': {'target_sensor': 'Sentinel-2'}}]}\n",
+      "[{'harmonize': {'target_sensor': 'Sentinel-2'}}]\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Clip images to the AOI's perimeter and harmonize the data with Dove Classic\n",
+    "tools = [\n",
+    "    planet.order_request.harmonize_tool('Sentinel-2')\n",
```

Comment:
FYI on [planet 2.0a6](https://pypi.org/project/planet/2.0a6/) the harmonize_tool function appears to not take any parameters, and defaults to PS2.

Reply:
Wow, great catch! I actually just merged [a PR](https://github.com/planetlabs/planet-client-python/pull/780), which addresses this issue. It's now on `main` and will be added to the next release.